ngrok-java-17/pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+        <groupId>com.ngrok</groupId>
+        <artifactId>ngrok-project</artifactId>
+        <version>v0.2.0</version>
```

Comment:
we should revert this version to `0.2.0-SNAPSHOT`, I think (also in the rest of the poms)